docs/gui/liquids.rst

```diff
@@ -2,41 +2,20 @@ gui/liquids
 ===========
 
 .. dfhack-tool::
-    :summary: Interactively paint liquids or obsidian onto the map.
-    :tags: untested fort armok map
+    :summary: Interactively paint liquids onto the map.
+    :tags: fort armok map
 
-This tool is a gui front-end to `liquids` and works similarly, allowing you to
-add or remove water/magma, and create obsidian walls & floors.
+This tool is a gui for spawning water or magma. Allowing you to click and paint
+liquid onto the map.
 
 .. warning::
 
-    There is **no undo support**. Be sure the settings are what you want before
-    hitting Enter!
-
-The :kbd:`b` key changes how the affected area is selected. The default
-:guilabel:`Rectangle` mode works by selecting two corners like any ordinary
-designation. The :kbd:`p` key cycles through modes for adding water, magma,
-obsidian walls/floors, or modifying liquid tile properties.
-
-When painting liquids, you can select the desired level with :kbd:`+`:kbd:`-`,
-and you can choose among setting it exactly, only increasing, or only decreasing
-with :kbd:`s`.
-
-In addition, :kbd:`f` allows disabling or enabling the flowing water
-computations for an area, and :kbd:`r` operates on the "permanent flow" property
-that makes rivers power water wheels even when full and technically not flowing.
-
-After setting up the desired operations using the described keys, use
-:kbd:`Enter` to apply them.
+    There is **no undo support**. Be sure you know what you are doing before
+    spawning any liquid!
 
 Usage
 -----
 
 ::
 
     gui/liquids
-
-Screenshot
-----------
-
-.. image:: /docs/images/liquids.png
```

Comment:
could you also remove this file?

Reply:
https://github.com/DFHack/dfhack/pull/2724